web/static/vendor/maps.js

```diff
@@ -0,0 +1,101 @@
+function initMap() {
```

Comment:
Ugh - so, I spent some time cleaning this up but don't want to go too crazy with this. Curious to know thoughts on it as it stands.